source/containers.tex

```diff
@@ -9200,6 +9238,665 @@
+    constexpr void clear() noexcept;
+
+    constexpr friend bool operator==(const inplace_vector& x,
+                                     const inplace_vector& y);
+    constexpr friend synth-three-way-result<T>
+      operator<=>(const inplace_vector& x, const inplace_vector& y);
+    constexpr friend void swap(inplace_vector& x, inplace_vector& y)
+      noexcept(N == 0 || (is_nothrow_swappable_v<T> &&
+                          is_nothrow_move_constructible_v<T>))
+      { x.swap(y); }
+  };
+};
+\end{codeblock}
+
+\rSec3[inplace.vector.cons]{Constructors}
+
+\indexlibraryctor{inplace_vector}
+\begin{itemdecl}
+constexpr explicit inplace_vector(size_type n);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{T} is \oldconcept{DefaultInsertable} into \tcode{*this}.
+
+\pnum
+\effects
+Constructs an \tcode{inplace_vector} with \tcode{n} default-inserted elements.
+
+\pnum
+\complexity
+Linear in \tcode{n}.
+\end{itemdescr}
+
+\indexlibraryctor{inplace_vector}
+\begin{itemdecl}
+constexpr inplace_vector(size_type n, const T& value);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{T} is \oldconcept{CopyInsertable} into \tcode{*this}.
+
+\pnum
+\effects
+Constructs an \tcode{inplace_vector} with \tcode{n} copies of \tcode{value}.
+
+\pnum
+\complexity
+Linear in \tcode{n}.
+\end{itemdescr}
+
+\indexlibraryctor{inplace_vector}
+\begin{itemdecl}
+template<class InputIterator>
+  constexpr inplace_vector(InputIterator first, InputIterator last);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Constructs an \tcode{inplace_vector} equal to the range \range{first}{last}.
+
+\pnum
+\complexity
+Linear in \tcode{distance(first, last)}.
+\end{itemdescr}
+
+\indexlibraryctor{inplace_vector}
+\begin{itemdecl}
+template<@\exposconcept{container-compatible-range}@<T> R>
+  constexpr inplace_vector(from_range_t, R&& rg);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Constructs an \tcode{inplace_vector} object with
+the elements of the range \tcode{rg}.
+
+\pnum
+\complexity
+Linear in \tcode{ranges::distance(rg)}.
+\end{itemdescr}
+
+\rSec3[inplace.vector.capacity]{Size and capacity}
+
+\indexlibrarymember{capacity}{inplace_vector}%
+\indexlibrarymember{max_size}{inplace_vector}%
+\begin{itemdecl}
+static constexpr size_type capacity() noexcept;
+static constexpr size_type max_size() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{N}.
+\end{itemdescr}
+
+\indexlibrarymember{resize}{inplace_vector}%
+\begin{itemdecl}
+constexpr void resize(size_type sz);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{T} is \oldconcept{DefaultInsertable} into \tcode{*this}.
+
+\pnum
+\effects
+%FIXME: Should "is \tcode{true}" be appended here?
+If \tcode{sz < size()},
+erases the last \tcode{size() - sz} elements from the sequence.
+Otherwise,
+appends \tcode{sz - size()} default-inserted elements to the sequence.
+
+\pnum
+\remarks
+If an exception is thrown, there are no effects on \tcode{*this}.
```

Comment:
```suggestion
If an exception is thrown, there are no effects.
```

Reply:
this is a normative change, please forward to LWG.
resolving here to reduce "noise"